go.mod

```diff
@@ -8,19 +8,20 @@ require (
 	github.com/google/go-sev-guest v0.7.0
 	github.com/google/go-tpm v0.9.0
 	github.com/google/logger v1.1.1
-	google.golang.org/protobuf v1.28.0
+	google.golang.org/protobuf v1.31.0
```

Comment:
You'll need to either update the protoc-gen-go version in .github/workflows/ci.yaml (I'm not sure if there's a github action available at this version) or re-generate these with the existing version. To do that, you'll need to have a local install at the old version and prefer its /bin location in your PATH environment variable.

Reply:
Only maintainers can modify the ci.yaml file.
I have changed the version of protoc-gen-go to 1.28.0 .

Reply:
It's still showing v1.31.0 as the version, have you changed it back to v1.28.0? the header in proto/attest/attest.pb.go is showing v1.28.0

You can either update both to 1.31, or just remain 1.28

Reply:
Done.